### PR TITLE
Feature/toast component

### PR DESCRIPTION
**Pull request by fabricewerger**

### CODE REVIEW

File: components/Toast/Toast.tsx [hunk 1]
```diff
@@ -41,56 +96,111 @@ const ToastComp = ({ variant = 'filled', label, title, disabled }: ToastProps) =
...
+          <time
+            className={`m-0 text-xs py-1 pl-3.5 break-all ${toastFamilyClasses['variant'][variant]['description']}`}
+            dateTime={eventDateRef.current.toISOString()}
+          >
+            {description}
           </time>
-        </Toast.Description>
-        <Toast.Action className="ToastAction" asChild altText="Goto schedule to undo">
+        </RadixToast.Description>
+        <RadixToast.Action className="ToastAction" asChild altText="Goto schedule to undo">
```
Comment (NathanClare) [inline components/Toast/Toast.tsx:191]: We might want to make the altText configurable (or generic "Close")

File: components/Toast/Toast.tsx [hunk 2]
```diff
@@ -41,56 +96,111 @@ const ToastComp = ({ variant = 'filled', label, title, disabled }: ToastProps) =
...
           }, 100)
         }}
       >
         {label}
       </button>
 
-      <Toast.Root
-        className="bg-neutral-white rounded-lg shadow p-4 grid gap-x-3.5 items-center data-[state=open]:transition"
+      <RadixToast.Root
+        duration={100000}
```
Comment (NathanClare) [inline components/Toast/Toast.tsx:174]: We may also want to pass this as a variable

Comment (fabricewerger) [reply components/Toast/Toast.tsx:137]: Added.

File: components/Toast/Toast.tsx (hunk 1 above)
Comment (fabricewerger) [reply components/Toast/Toast.tsx:154]: Added both a generic close and the option to change it in storybook.

**Pull request merged**